recipes/sup.rb

```diff
@@ -0,0 +1 @@
+hab_sup 'default'
```

Comment:
yes, we can't port that into core chef, and its very 'leftpad':

http://www.haneycodes.net/npm-left-pad-have-we-forgotten-how-to-program/